src/apps/loyalty/client_side_analytics.ts

```diff
@@ -0,0 +1,3 @@
+import { initAnalytics } from "./analytics"
+
+initAnalytics()
```

Comment:
Actually, I think there is a need for a separate file, for webpack to bundle. Otherwise, we just have a big `analytics.ts` lib file, but we need a file that actually invokes the appropriate methods. So can I just leave this as-is?